doc/servermd/Server.md

```diff
@@ -446,9 +447,9 @@ To stop the MCU cluster, follow these steps:
         cd Release-<Version>/
         bin/daemon.sh stop [portal/conference-agent/webrtc-agent/streaming-agent/audio-agent/video-agent/recording-agent/sip-agent/sip-portal]
 
-### 2.3.12 MCU cluster’s fault tolerance / resilience {#Conferencesection2_3_12}
+### 2.4 MCU cluster’s fault tolerance / resilience {#Conferencesection2_3_12}
```

Comment:
MCU -> ICS Server

Reply:
I got too many commits in a single PR,
can you reduce the commit message size when squash them into one commit